templates/e-commerce/api/high-low-all-products.json

```diff
@@ -1,88 +1,88 @@
 {
```

Comment:
Comments are not supported in JSON. I tried different forms of JS comments anyway but it broke the page. I've added a file overview to build.js instead.

I've read suggestions on adding a __comments field to the beginning of the object, but I don't think this is an elegant solution. Did you have something else in mind or can we stick to the build.js comment?